### PR TITLE
Command to login to locally running instance

### PR DESCRIPTION
**Pull request by efirs**
Allow to easily point all subsequent Tigris commands to the locally running instance.

Usage:

```
tigris login dev
tigris login localhost
tigris login localhost:9091
```

It clears credentals in the config file and sets URL to localhost.

### CODE REVIEW

File: cmd/login.go [hunk 1]
```diff
@@ -354,14 +353,41 @@ var loginCmd = &cobra.Command{
   if are not already signed in to the account
 * You'll see "Successfully authenticated" on success
 * You can now return to the terminal and start using the CLI`,
-	Example: `tigris login api.preview.tigrisdata.cloud`,
+	Example: `
+# Login to the hosted platform
+tigris login api.preview.tigrisdata.cloud
+
+# Point all subsequent commands to locally running instance
+tigris login local
```
Comment (ovaistariq) [inline cmd/login.go:361]: Since the local Tigris instance is interacted with by using the term `dev`, can we change this to be `dev` instead of `login` ?

Comment (efirs) [reply cmd/login.go:361]: `tigris login dev` you mean?

Comment (ovaistariq) [reply cmd/login.go:361]: Yup

Comment (efirs) [reply cmd/login.go]: Done.

**Review by ovaistariq — DISMISSED** [review]

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**